### DISCUSSION TITLE
Minio releases of RELEASE.2021-05 have been removed from Docker hub

### DISCUSSION THREAD

**Discussion by shakibamoshiri** [minio/minio#13479] [Q&A] [answered]
Minio releases of RELEASE.2021-05 have been removed from Docker hub while other tags before and after (**RELEASE.2021-04 && RELEASE.2021-06**) are still available .
Is there any issue or particular reason for this ?
missing: https://hub.docker.com/r/minio/minio/tags?page=1&name=RELEASE.2021-05
Thanks (edited)

**Comment by harshavardhana** [comment, marked as answer]
Yes it has been removed as they were quarantined check why

https://github.com/minio/minio/releases/tag/RELEASE.2021-09-09T21-37-07Z

```
NOTE: It is advised that anyone upgrading from any release from 2020 should directly upgrade to
RELEASE.2021-06-07T21-40-51Z - (or any release prior to RELEASE.2021-05-11T23-27-41Z so that
you can then directly upgrade to RELEASE.2021-06-07T21-40-51Z or later). A healing issue that leads
to premature truncation of objects when healing legacy content before versioning was introduced
refer #12459, #12410, and #12451
```

To avoid accidental usage of those containers and binaries they were quarantined.

**Reply by shakibamoshiri** [reply]
@harshavardhana I know your answer is right and have no problem **Mark it as answer** just needed to know which stable version we can directly upgrade to. We are afraid of data loss.

**Comment by shakibamoshiri** [comment]
Thank you so much.
Would mind telling me for `RELEASE.2021-05-20T22-31-44Z` which version is best one to **directly** upgrade to?
The `RELEASE.2021-06-07T21-40-51Z` or any closest one(s) which is (are) stable?

**Reply by harshavardhana** [reply]
> Thank you so much. Would mind telling me for `RELEASE.2021-05-20T22-31-44Z` which version is best one to **directly** upgrade to? The `RELEASE.2021-06-07T21-40-51Z` or any closest one(s) which is (are) stable?

Latest @k-five
